src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -4,34 +4,64 @@
 import java.math.RoundingMode;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 public class LottoMachine {
-    private final int buyLottoPrice;
 
+    private final static int LOTTO_PRICE = 1000;
```

Comment:
`static` 키워드를 `final` 키워드보다 먼저 작성하는 것이 컨벤션입니다. :)
> https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers

```suggestion
    private static final int LOTTO_PRICE = 1000;
```